### ISSUE TITLE
Has this ever run successfully under ROS2?

### ISSUE THREAD

**Issue by JesseCake** [flynneva/ros2serial_python#3]
The more I dig into this, the more it looks like this isn't functional. Is that right?

**Comment by flynneva** [comment]
@JesseCake nope, this was a side project I thought I needed a few years ago but then realized I didn't so I never fully completed the port. Sorry!
